helm/hpcc/values.schema.json

```diff
@@ -1092,7 +1113,10 @@
         "labels": {
           "type": "object",
           "additionalProperties": { "type": "string" }
-        }
+        },
+	"ingress": {
```

Comment:
minor: indentation issue, tabs vs spaces?

Reply:
Is this the right place for the roxie ingress section, or should it go under roxie.services under each "roxieservice" ?

Reply:
Current roxie networkpolicy rules are under roxie instead of roixe.services

Reply:
fixed the indentation